cli/Makefile

```diff
@@ -32,14 +32,21 @@ go-turbo$(EXT): $(GENERATED_FILES) $(SRC_FILES) go.mod turborepo-ffi-install
 
 .PHONY: turborepo-ffi-install
 turborepo-ffi-install: turborepo-ffi
-	cp -r ../crates/turborepo-ffi/bindings.h \
-	      ../crates/turborepo-ffi/target/release/libturborepo_ffi.a \
-	      ./internal/ffi
+	cp ../crates/turborepo-ffi/bindings.h ./internal/ffi
+	cp ../crates/turborepo-ffi/target/release/libturborepo_ffi.a ./internal/ffi/libturborepo_ffi_$(UNAME).a
+
+.PHONY: turborepo-ffi-install-win
+turborepo-ffi-install-win: turborepo-ffi-win
+	cp ../crates/turborepo-ffi/bindings.h ./internal/ffi
+	cp ../crates/turborepo-ffi/target/release/libturborepo_ffi.a ./internal/ffi/libturborepo_ffi_Windows.a
```

Comment:
If we cross compile for windows it outputs a `libturborepo_ffi.a` file instead of a `turborepo_ffi.lib` file?

Reply:
That seems to be the case on my machine...

Reply:
Doesn't happen when compiling natively though